src/Fixer/PhpUnit/PhpUnitNoExpectationAnnotationFixer.php

```diff
@@ -263,8 +263,9 @@ private function extractContentFromAnnotation(Annotation $annotation)
     {
         $tag = $annotation->getTag()->getName();
 
-        Preg::match('/@'.$tag.'\s+(.+)$/s', $annotation->getContent(), $matches);
-        $content = $matches[1];
+        $content = Preg::match('/@'.$tag.'\s+(.+)$/s', $annotation->getContent(), $matches)
+            ? $matches[1]
```

Comment:
Can we check the `$matches` size and return an empty string for no match? There is no point to assing empty string to `$content` and execute `preg_match` on it.

Reply:
@kubawerlos You mean something like these lines?:

```php
    /**
     * @param Annotation $annotation
     *
     * @return string
     */
    private function extractContentFromAnnotation(Annotation $annotation)
    {
        $tag = $annotation->getTag()->getName();

        if (
            Preg::match('/@'.$tag.'\s+(.+)$/s', $annotation->getContent(), $matches)
            && Preg::match('/\R/u', $matches[1])
        ) {
            return rtrim(Preg::replace('/\s*\R+\s*\*\s*/u', ' ', $matches[1]));
        }

        return '';
    }
```

It does not set the variable at all.

Reply:
@ktomk I meant like this:
```php
    private function extractContentFromAnnotation(Annotation $annotation)
    {
        $tag = $annotation->getTag()->getName();

        if (1 !== Preg::match('/@'.$tag.'\s+(.+)$/s', $annotation->getContent(), $matches)) {
            return '';
        }

        $content = $matches[1];

        if (Preg::match('/\R/u', $matches[1])) {
            return Preg::replace('/\s*\R+\s*\*\s*/u', ' ', $matches[1]);
        }

        return rtrim($content);
    }
```

Also, I wouldn't call it refactoring, I think of it as Boy Scout Rule.

Reply:
looks better IMHO, I pushed the change in